libvast/src/system/application.cpp

```diff
@@ -273,10 +273,10 @@ auto make_spawn_source_command() {
       .add<caf::atom_value>("batch-encoding", "encoding type of table slices "
                                               "(arrow or msgpack)")
       .add<size_t>("batch-size", "upper bound for the size of a table slice")
-      .add<std::string>("batch-timeout", "timoeut after which batched table "
-                                         "slices are forwarded")
-      .add<size_t>("max-events,n", "the maximum number of events to "
-                                   "import"));
+      .add<std::string>("batch-timelimit", "time limit after which batched "
+                                           "table slices are forwarded")
```

Comment:
Why did you rename this from timeout to timelimit? The latter is a made up word.

```suggestion
      .add<std::string>("batch-timeout", "timeout after which batched table "
                                         "slices are forwarded")
```

Reply:
Note that this must be applied all throughout the PR.

Reply:
I missed those sites. `batch-timelimit` was the first naming approach when I split the functionality out from `batch-timeout`.